harry-olabs/avi-hub
# AVI-29: Enhancement to layout component

### What does this PR do


Makes adjustment to the UI layout component with huge enhancements to the logic.

Closes #29 

### How should it be tested



- Spin up the app in dev mode with `npm start`
- Navigate to Dashboard and Manage Users page
- App should run fine with smooth navigation to pages

### Screenshots



| Screenshot-01 | Screenshot-02 |
|----------------|----------------|
|  <img width="1382" alt="Screenshot 2023-08-02 at 11 57 46" src="https://github.com/harry-olabs/avi-hub/assets/135703492/616cef88-f7bd-447f-93fc-70b48dc0f14f"> | <img width="1381" alt="Screenshot 2023-08-02 at 11 58 03" src="https://github.com/harry-olabs/avi-hub/assets/135703492/132d9e95-2125-4c01-a5d5-13fc11a74327"> |
